common/lib/xmodule/xmodule/tests/test_graders.py

```diff
@@ -97,6 +98,14 @@ def __init__(self, graded_total, display_name):
             self.graded_total = graded_total
             self.display_name = display_name
 
+        @property
+        def percent_graded(self):
+            if self.graded_total.possible > 0:
+                # Rounds to two decimal places.
+                return around(self.graded_total.earned / self.graded_total.possible, decimals=2)
+            else:
+                return 0.0
```

Comment:
Please update this to call `compute_percent` in `scores.py` to avoid duplication of the logic.  If you get an import error, you may need to import `scores.py` within `percent_graded` implementation.  Eventually, `graders.py` and `test_graders.py` should move into the `grades` djangoapp.

Reply:
Done.